### ISSUE TITLE
MDI forms don't maximize properly when Ribbon is present

### ISSUE THREAD

**Issue by Ahmed-Abdelhameed** [Cocotteseb/Krypton#5]
_Note: This issue doesn't occur with the original Krypton library (v. 4.5), it only occurs with this fork which I'm using exclusively for the Outlook Grid._

- The issue basically occurs when having an MDI container form with a Ribbon control, and then try to maximize the MDI child.
- The behavior of the issue: When maximizing the MDI child form, it will appear as if it's docked right under the ribbon _(the title bar of the child form appears underneath the ribbon)_ until the parent form is resized.
- The easiest way to reproduce the bug is to have the parent form (the MDI container) maximized before maximizing the child form. _It can also occur while the parent form isn't maximized under some conditions_. 

So, basically, it appears like this:

![ribbon](https://user-images.githubusercontent.com/18140178/33684048-8cca05dc-dad5-11e7-92ae-3486058cc4e4.png)

..instead of this:

![ribbon2](https://user-images.githubusercontent.com/18140178/33684056-968a3fa6-dad5-11e7-959b-33f8de77661b.png)
